apex_rostest/apex_launchtest/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[coverage:run]
```

Comment:
Hmm, we don't do coverage checks, do we? Just wondering if it'd make sense to bring that improvement along with the repo.

Reply:
We don't collect coverage stats right now in our CI, but it would be nice to have this at some point. I don't know, however, if putting it in the setup.cfg makes sense, as it should be possible to run it with and without. I don't know what the recommended strategy is.

Reply:
My understanding is that a bunch of different tools may look for a setup.cfg file and should each ignore configuration sections not meant for them.  If this is causing a problem with another tool that expects its setup.cfg file to be free of `[coverage:run]` sections, we could also move it to a file called `.coveragerc`

See https://coverage.readthedocs.io/en/v4.5.x/config.html for how 'coverage' uses configuration files